res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -18,20 +18,24 @@ limitations under the License.
     display: flex;
     flex-direction: column;
 
+    padding-right: 0;
+
     .mx_BaseCard_header {
-        padding: 6px 0;
+        padding: 6px 8px 6px 0;
 
-        .mx_BaseCard_close {
+        .mx_BaseCard_close,
+        .mx_BaseCard_back {
             margin-top: 15px;
         }
-    }
 
-    .mx_AccessibleButton.mx_BaseCard_back {
-        display: none;
+        .mx_BaseCard_close {
+            right: -8px;
+        }
     }
 
-    &__header {
-        width: calc(100% - 40px);
+    .mx_ThreadPanel__header {
```

Comment:
Hmm, not used to seeing this `__` style in Element Web, but I assume you know what you're doing...